APSHTTPClient/APSHTTPRequest.m

```diff
@@ -342,8 +240,7 @@ -(void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didRecei
         if(self.requestPassword != nil && self.requestUsername != nil) {
             handled = YES;
             NSURLCredential *credential = [NSURLCredential credentialWithUser:self.requestUsername
-                                                                     password:self.requestPassword
-                                                                  persistence:NSURLCredentialPersistenceForSession];
```

Comment:
Can you explain the line change #346? The default is `NSURLCredentialPersistenceNone` which will now be used. If using `NSURLCredentialPersistenceForSession` fails now, it should be fixed in a different place, in this case the appcelerator.https module.

Reply:
I am not sure why it is showing like this here. The actual code, if you see in file is -
NSURLCredential * credential = [NSURLCredential credentialWithUser:self.requestUsername
                                                                     password:self.requestPassword                                                                  persistence:NSURLCredentialPersistenceForSession];

Reply:
Haha. It's the formatting that makes it look like [this](https://github.com/vijaysingh-axway/APSHTTPClient/blob/2195b23ea0ebd6bddb5545988b4d501eaca92a76/APSHTTPClient/APSHTTPRequest.m#L242). Please just break the line and it will be fine.